cmd/wallet-sdk-gomobile/api/api.go

```diff
@@ -65,14 +65,13 @@ type CredentialWriter interface {
 }
 
 // Crypto defines useful Crypto operations.
-// TODO: Define more precisely the input and output formats.
 type Crypto interface {
-	// Sign will sign msg using a matching signature primitive in key referenced by keyID
+	// Sign will sign msg using a matching signature primitive from key referenced by keyID
```

Comment:
I'm still not sure what this means... how do you get a signature primitive from a key?

Reply:
Crypto will get a signature primitive from keyID. Crypto and KMS implementation are always created as a pair. So Crypto implementation will know how to get signature primitive